src/sdk/pynni/nni/compression/torch/speedup/compressor.py

```diff
@@ -28,16 +28,22 @@ def get_module_by_name(model, module_name):
     """
     name_list = module_name.split(".")
     for name in name_list[:-1]:
-        model = getattr(model, name)
-    leaf_module = getattr(model, name_list[-1])
-    return model, leaf_module
+        if hasattr(model, name):
+            model = getattr(model, name)
+        else:
+            return None, None
+    if hasattr(model, name_list[-1]):
+        leaf_module = getattr(model, name_list[-1])
+        return model, leaf_module
+    else:
+        return None, None
```

Comment:
curious about what kind of `module_name` returns `None, None`?

Reply:
Those functional node which  is not an attribute of model object,  such as `relu`, `add`